lambdas/shared/setup.py

```diff
@@ -0,0 +1,17 @@
+from setuptools import setup
```

Comment:
this file could use a comment explaining intended use and that /lambdas/whatever will expect that `pip install .` has been called

Reply:
Eh, that's like literally what `setup.py` is for :)

Reply:
sorry, my logic was backwards. it's not about this directory but about lambdas/not_shared. I think a lambdas/README.md is worthwhile to mention this, but not religious. people will work in isolated folders and it's not super obvious that an external setup file is needed.